crates/astria-composer/src/executor/mod.rs

```diff
@@ -94,7 +99,19 @@ pub(crate) use builder::Builder;
 const BUNDLE_DRAINING_DURATION: Duration = Duration::from_secs(16);
 
 type StdError = dyn std::error::Error;
-
+#[derive(Debug, thiserror::Error)]
+pub(crate) enum EnsureChainIdError {
+    // <other error variants from getting genesis, etc>
```

Comment:
remove that comment :) That was just for not typing out everything in my review.

Reply:
Done in 33ae83d.